### PR TITLE
(#1192) Adds Typography Page content to Docs Site

### LINKED ISSUE #1192: Foundations subpage: Typography
Update the Typography page within the Foundations section of the documentation site with the content needed for NCIDS for All pilot launch.

### Tasks
- [x] Create content in the shared [Word Document](https://nih.sharepoint.com/:w:/r/sites/NCI-OCPLNewCgovUXDesignSystem/_layouts/15/Doc.aspx?sourcedoc=%7B57365813-2449-474C-99A2-877928E8BBF4%7D&file=Typography.docx&wdLOR=cEAB2BF9E-3301-0444-AB1D-AECBA05A746C&action=default&mobileredirect=true)
    - [x] IA responsibility:
    - [x] Dev responsibility:
    - [x] Design responsibility:
- [x] Review content in shared document
- [x] Create images in shared folder: `<insert link to shared folder>`
- [x] Convert content and images to markdown
- [ ] Add to documentation site 

[Typography.txt](https://github.com/NCIOCPL/ncids/files/12860979/Typography.txt)
[Font Size and Family.txt](https://github.com/NCIOCPL/ncids/files/12861213/font_size_family_markdown.txt)

Needed images are here:

Poppins: poppins-sample
Roboto: roboto-mono_sample
Open Sans: open-sans_sample

### Resources
- [Doc Site Content Inventory](https://nih.sharepoint.com/:x:/r/sites/NCI-OCPLNewCgovUXDesignSystem/Shared%20Documents/General/Details%20-%20NCIDS%20Pilot/Documentation%20Site/Doc%20Site%20Content%20Inventory.xlsx?d=wff83f29cd1554e99892fd963404c69a1&csf=1&web=1&e=QZgQLq)
- [Markdown for Typography](https://nih.sharepoint.com/sites/NCI-OCPLNewCgovUXDesignSystem/Shared%20Documents/Forms/AllItems.aspx?id=%2Fsites%2FNCI%2DOCPLNewCgovUXDesignSystem%2FShared%20Documents%2FGeneral%2FDetails%20%2D%20NCIDS%20Pilot%2FDocumentation%20Site%2FPage%20Content%20%2D%20Markdown%2FTypography%2Etxt&parent=%2Fsites%2FNCI%2DOCPLNewCgovUXDesignSystem%2FShared%20Documents%2FGeneral%2FDetails%20%2D%20NCIDS%20Pilot%2FDocumentation%20Site%2FPage%20Content%20%2D%20Markdown)
- [Markdown for Font Size and Family](https://nih.sharepoint.com/sites/NCI-OCPLNewCgovUXDesignSystem/Shared%20Documents/Forms/AllItems.aspx?id=%2Fsites%2FNCI%2DOCPLNewCgovUXDesignSystem%2FShared%20Documents%2FGeneral%2FDetails%20%2D%20NCIDS%20Pilot%2FDocumentation%20Site%2FPage%20Content%20%2D%20Markdown%2Ffont%5Fsize%5Ffamily%5Fmarkdown%2Etxt&parent=%2Fsites%2FNCI%2DOCPLNewCgovUXDesignSystem%2FShared%20Documents%2FGeneral%2FDetails%20%2D%20NCIDS%20Pilot%2FDocumentation%20Site%2FPage%20Content%20%2D%20Markdown)

### PR DESCRIPTION
**Pull request by dlescarbeau**
Closes #1192

### CODE REVIEW

File: docs/content/foundation/typography/index.mdx [hunk 1]
```diff
@@ -6,8 +6,38 @@ nav_order: 40
...
+
+## Roboto Mono
+
+Roboto Mono is only used in code for any place that uses the mono style
+
+![](../../documentation-images/roboto-example.png)
+
+## Custom NCIDS Type Scale
+
+Insert the Theme Token table from the following page: https://designsystem-dev.cancer.gov/develop/foundation/typography/fonts#theme-tokens (xs-3xl “$theme-type-scale”)
```
[the remark this answers is not in the document]
Comment (dlescarbeau) [reply docs/content/foundation/typography/index.mdx]: _Oops_

**Comment by bennettcc** [comment]
Approved by product on 10/5

File: docs/content/foundation/typography/index.mdx [hunk 2]
```diff
@@ -4,10 +4,55 @@ nav_label: Typography
...
+
 # Typography
 
-Nulla facilisis vitae lectus quis ultricies. Aliquam sodales scelerisque sapien et dictum.
-Mauris eu nunc sit amet lorem blandit gravida et ac dui. Suspendisse accumsan eros sit
-amet arcu lobortis, in consequat nunc molestie. Aenean lobortis molestie convallis.
-Nullam sit amet turpis vel nisi tristique dapibus sed in libero.
-Vivamus ac tristique nisl, eu bibendum eros.
+Typography helps establish hierarchy which functions to present content clearly and efficiently to users by creating clear visual patterns. NCIDS supports 3 Google fonts, which are easy for designers to embed into Figma and developers to utilize in code. You can download these fonts from Google fonts:
+
```
[the remark this answers is not in the document]
Comment (olitharp-nci) [reply docs/content/foundation/typography/index.mdx]: I agree this belongs on this page and not in /foundations/typography/fonts

@sarinapadilla is confirming

we're probably have you yank it out of the other page so it lives here

**Review by andyvanavery31 — APPROVED** [review]
Passes Product review.

**Review by olitharp-nci — APPROVED** [review]

**Pull request merged**